db/seeds.rb

```diff
@@ -102,6 +102,14 @@ def sample_numeric_grade_from_letter(grade_letter)
   end
 end
 
+Student.all.each do |student|
+  StudentPhoto.create!(
+    student_id: student.id,
+    file_digest: Digest::SHA256.hexdigest(student.local_id),
+    file_size: 37010,
+    s3_filename: 'MissingAvatar.png'
```

Comment:
This code seems like it's simulating "there is a photo for this student!" which is awesome.  But using a "missing avatar" image makes it seem like that's a feature - that the endpoint will show this if the server can't find an image for them.  Maybe consider changing the image to a cartoon image of a student to clarify?

Reply:
👍 — that makes sense... I'll change the filename at the very least.